src/sentry/snuba/tasks.py

```diff
@@ -113,9 +118,12 @@ def delete_subscription_from_snuba(query_subscription_id, **kwargs):
 
 
 def build_snuba_filter(dataset, query, aggregate, environment, params=None):
+    def alerts_resolve_column(col):
+        return resolve_column(col, Dataset.Events)
```

Comment:
Won't you also need to go on to the transactions dataset sometimes?

Reply:
Yeah this part is a WIP, I don't _fully_ understand our needs yet but I think I'll just be forking between Events and Transactions here - but I'd need to add that dataset to the dict it seems (but it has fields defined in Columns so thats good).

Reply:
I don't think we'll need to do any extra translations on the transaction dataset. https://github.com/getsentry/snuba/blob/b639cbc5a38dc7b9a6083d029860914518482f81/snuba/datasets/transactions.py#L72

We only have this problem with events because we were unable to move the translations out of the discover dataset into the events dataset.

We might still need to run them through the Discover dataset translations though? Not totally sure.